_posts/2025-06-06-#3736.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: pr
+date: 2025-05-09
+title: "Transaction negotiation during splicing"
+pr: 3736
+authors: [ optout21 ]
+components: [ "Splicing", "Channel Funding", "Transaction Negotiation"]
+host: vincenzopalazzo
+status: upcoming
+commit:
+---
+
+## Notes
+[Splicing] is a process that allows users to change the funding transaction of a channel without
+closing it. This PR introduces the concept of "Transaction Negotiation" for splicing, which is a way
+to construct transactions interactively between peers using [INTERACTIVE TRANSACTION CONSTRUCTION].
+
+
+## Questions
+
+1. Have you reviewed [ldk/3736]? If so, is it a [Concept ACK, Approach ACK, Tested ACK, or NACK?](https://github.com/lightningdevkit/rust-lightning/blob/master/CONTRIBUTING.md#peer-review)
+2. Define "Transaction Negotiation" in the context of splicing. What is its purpose and how it
+   differs from regular "Interactive Transaction Construction"?
+3. When does "Transaction Negotiation" start and when does it end? What are the key steps
+   involved?
+4. [ldk/3736] introduces `Channel::as_renegotiating_channel`. How is this method different from
+   `Channel::as_negotiating_channel`?
```

Comment:
These are actually not on `Channel`, but on `FundedChannel` and `PendingV2Channel`, respectively. The PR introduces both methods, too.